doc/source/serve/multi-app.md

```diff
@@ -78,66 +81,80 @@ $ serve deploy config.yaml
 > Sent deploy request successfully!
 ```
 
-Query the applications at their respective endpoints, `/calculator` and `/greet`.
+Query the applications at their respective endpoints, `/classify` and `/translate`.
 ```pycon
->>> requests.post("http://localhost:8000/calculator", json=["ADD", 5]).json()
-7
+>>> requests.post("http://localhost:8000/classify", json={"image_url": "https://cdn.britannica.com/41/156441-050-A4424AEC/Grizzly-bear-Jasper-National-Park-Canada-Alberta.jpg"}).text
+'brown bear, bruin, Ursus arctos'
 
->>> requests.post("http://localhost:8000/greet", json="Bob").json()
-'Good morning Bob!'
+>>> requests.post("http://localhost:8000/translate", json={"text": "Hello, the weather is quite fine today!"}).text
+'Hallo, das Wetter ist heute ziemlich gut!'
 ```
 
-:::{tip}
-If you prefer to use `cURL` to ping these endpoints, add the `-L` flag. The `DAGDriver` does an HTTP redirect, and the flag ensures that `cURL` follows the redirect:
-
-```
-curl -L -X POST -H "Content-Type: application/json" -d '["ADD", 5]' localhost:8000/calculator
+#### Development Workflow with `serve run`
+You can also use the CLI command `serve run` to run and test your application easily, either locally or on a remote cluster. 
+```console
+$ serve run config.yaml
+> 2023-04-04 11:00:05,901 INFO scripts.py:327 -- Deploying from config file: "config.yaml".
+> 2023-04-04 11:00:07,505 INFO worker.py:1613 -- Started a local Ray instance. View the dashboard at http://127.0.0.1:8265
+> 2023-04-04 11:00:09,012 SUCC scripts.py:393 -- Submitted deploy config successfully.
 ```
+
+The command `serve run` blocks the terminal, which allows logs from Serve to stream to the console. This helps you test and debug your applications easily. If you want to change your code, you can hit Ctrl-C to interrupt the command and shutdown Serve and all its applications, then rerun `serve run`.
+
+:::{note}
+`serve run` only has support for running multi-application config files. If you want to run applications directly without a config file `serve run` can only run one application at a time.
```

Comment:
I think this note is a little unclear. serve run does allow you to specify applications right? Maybe explain that when using serve run it'll restart everything

Reply:
`serve run` can't run multiple applications by import path. I've updated the note to make that more clear!